internal/validations/validations.go

```diff
@@ -60,6 +79,17 @@ func IsValidImageURI(imageURI string) bool {
 	return err == nil
 }
 
+// IsApprovedImageURI determines if the image URI is approved for use. Default is allow-all. If any container_uris are
+// set in the config, then provided input must be a direct match.
+func IsApprovedImageURI(imageURI string) bool {
```

Comment:
This is more about placement, but I _think_ (don't quite remember – would have to look more) `./internal/validations` was intended for shared validations/helpers across server/cli.

Reply:
Noted. Happy to migrate if needed. I literally just walked the path for where API validation funcs were currently and piggy-backed on.

Reply:
Looked into migrating to a separate package nested within service, but ran into a number of snags because of the general layout of the repo and the separation created from cli/request/response/service. Can discuss on slack, but short version is it created some ugly ripples to make things work, and didn't seem to bring any benefits. Parking it here for the moment to avoid additional ripples in the context of this one change.